### ISSUE TITLE
Can't center align Social Embeds

### ISSUE THREAD

**Issue by timomak** [justinmahar/react-social-media-embed#4]
I already submitted a ticket saying I can't change the width of the Social Embeds, hence I can't make it fill a Div to '100%'. 

When I wrap the Social Embed into a `<div>`, I can't get it to align in the center. 

Only TikTok seems to auto-align and fill the screen. Although the TikTok Embed is narrow and it has a white background color behind the video itself that fills the full width of the parent container.

**Comment by justinmahar** [comment]
We might be able to get around this by setting a width on the embed and putting it in a div with display flex and justify content center. I'll play around with sizing a bit and see if I can get things working.

**Comment by timomak** [comment]
<img width="537" alt="Screen Shot 2022-02-18 at 9 45 07 AM" src="https://user-images.githubusercontent.com/22608654/154735161-0553adb9-bef7-4139-9c97-c1b0a6d72500.png">

Just to be clear, this is an example of what I'm seeing.

I have a parent container with a red background color and two embeds inside.

`<div>
    <TwitterEmbed
        width={'100%'}
        url='https://twitter.com/PixelAndBracket/status/1356633038717923333'
      />
      <TikTokEmbed
        width={'100%'}
        url='https://www.tiktok.com/@epicgardening/video/7055411162212633903?is_copy_url=1&is_from_webapp=v1'
      />
</div>
`

Look at how the TikTok does fill the full width, but it does with the parent container holding the TikTok iframe I'm assuming.

**Comment by justinmahar** [comment]
@timomak TikTok is a little tricky because the iframe has a white background. This is TikTok, not me. But I was able to work around that by surrounding the embed in a div, centering the contents, and sizing the embed to a width of 325. I also added a border radius to the div, too, to match TikTok's. This seems to have done the trick.

<img width="1915" alt="Screenshot at Feb 18 20-33-10" src="https://user-images.githubusercontent.com/3472836/154780781-37aafe90-5013-4b79-8722-a566595cd6cc.png">
